wrf/utils.py

```diff
@@ -439,9 +457,6 @@ def to_dataframe(self,start_time,
         unstagger: bool, optional
```

Comment:
I don't quite follow what this is doing. The way things have been until recently, tslist outputs w and z on full levels and the rest on half levels. Now, if tslist_unstagger_winds = .true., then everything is on half levels, and if it is false, then it is the same as before. I'm not sure if this takes that into consideration or what we'd need to do to make sure that we're not unstaggering variables that are unstaggered already.

Reply:
I'll have to double check that this does what I think it's doing.

Reply:
So what I had originally done was to have the staggered vars be `Tower._create_datadict(...,staggered_vars=['w','ph'])` by default. Then after our discussion, @phawbeck, you discovered that w was actually already unstaggered automatically, regardless of the WRF namelist input. So instead of deleting all of the extra functionality I wrote, I changed the defaults to `_create_datadict(...,unstagger=False,staggered_vars=['ph'])`.